Source/IGListSingleSectionController.h

```diff
@@ -52,8 +55,28 @@ IGLK_SUBCLASSING_RESTRICTED
  (usually "self") or the IGListAdapter. Pass in locally scoped objects or use weak references!
  */
 - (instancetype)initWithCellClass:(Class)cellClass
-                   configureBlock:(void (^)(id item, __kindof UICollectionViewCell *cell))configureBlock
-                        sizeBlock:(CGSize (^)(id<IGListCollectionContext> collectionContext))sizeBlock NS_DESIGNATED_INITIALIZER;
+                   configureBlock:(IGListSingleSectionCellConfigureBlock)configureBlock
+                        sizeBlock:(IGListSingleSectionCellSizeBlock)sizeBlock NS_DESIGNATED_INITIALIZER;
+
+/**
+ Create a new section controller for a given nib name, bundle and cell type that will always have only one cell when present in a feed.
+ 
+ @param nibName        The name of the nib file for the single cell.
+ @param bundle         The bundle in which to search for the nib file.
+ @param cellClass      The UICollectionViewCell subclass for the single cell.
+ @param configureBlock A block that configures the cell with the item given to the section controller.
+ @param sizeBlock      A block that returns the size for the cell given the collection context.
+ 
+ @return A new section controller.
+ 
+ @warning Be VERY CAREFUL not to create retain cycles by holding strong references to: the object that owns the adapter
+ (usually "self") or the IGListAdapter. Pass in locally scoped objects or use weak references!
+ */
+- (instancetype)initWithNibName:(NSString *)nibName
+                         bundle:(NSBundle * _Nullable)bundle
+                      cellClass:(Class)cellClass
+                 configureBlock:(IGListSingleSectionCellConfigureBlock)configureBlock
+                      sizeBlock:(IGListSingleSectionCellSizeBlock)sizeBlock;
```

Comment:
Ya, let's break it. @svenbacia mind changing this so there's only a single init? Note that this will break the sample apps and unit tests, but chasing the compiler errors should be easy.